app/src/forms/admin/service.js

```diff
@@ -27,7 +51,7 @@ const service = {
       // grab all users that have roles on this form...
       const users = [...new Set(accessItems.filter(x => x.formId === formId && x.roles.length))];
       const form = accessItems.find(x => x.formId === formId);
-      const userRoles = users.map(x => { return {userId: x.userId, username: x.username, roles: x.roles}; });
+      const userRoles = users.map(x => { return { userId: x.userId, username: x.username, roles: x.roles }; });
```

Comment:
The map callback function can be more cleanly defined using a closure to avoid needing a return statement (recommended for direct object returns or one liners):
```js
const userRoles = users.map(x => ({ userId: x.userId, username: x.username, roles: x.roles }));
```
If you really wanted to be fancy... could also use Object Destructuring and Property Shorthand:
```js
const userRoles = users.map(({ userId, username, roles }) => ({ userId, username, roles })(object));
```
Ref: https://stackoverflow.com/a/39333479

For readability's sake though, I'd suggest the simple closure option over the destructuring one.

Reply:
Yeah first just looks more readable to me